src/model/resources.py

```diff
@@ -34,8 +35,10 @@ def model_get_modified(model_id):
 
     try:
         model_wrapper = storage.get(model_id)
-    except KeyError:
+    except ModelNotFound:
```

Comment:
With my suggested change above this could become:
```python
except HTTPError as error:
    return error.reason, error.code
```
Which we could use to handle 401, 403, and 404. Given that the exception is properly instantiated by `storage.get`.